src/main/plugins/org.dita.html5/xsl/simpletable.xsl

```diff
@@ -216,7 +217,17 @@ See the accompanying LICENSE file for applicable license.
   <xsl:template match="*[contains(@class, ' topic/stentry ')]" mode="simpletable:entry">
     <xsl:apply-templates select="." mode="table:common"/>
     <xsl:apply-templates select="." mode="headers"/>
-    <xsl:apply-templates/>
+    <xsl:choose>
+      <xsl:when test="*|text()|processing-instruction()">
+        <xsl:apply-templates/>
+      </xsl:when>
+      <xsl:when test="@specentry">
+        <xsl:apply-templates select="@specentry"/>
+      </xsl:when>
+      <xsl:otherwise>
+        <xsl:text>&#160;</xsl:text>
```

Comment:
Two reasons really, but I don't feel very strongly about either of them. 

First - it was there in the XHTML version where this originated - so technically it is restoring the function that was there in HTML5 before the code base split.

Second - it is there in XHTML because in the past, we saw a lot of browser issues where the overall formatting of the table and document were really messed up by `<td/>`, as opposd to `<td></td>` or `<td>&nbsp;</td>`. I don't know if that is relevant any more.